test/dtypes.c

```diff
@@ -7567,6 +7567,177 @@ test_named_indirect_reopen(hid_t fapl)
     return 1;
 } /* end test_named_indirect_reopen() */
 
+/*-------------------------------------------------------------------------
+ * Function:    test_named_indirect_reopen_file
+ *
+ * Purpose: Tests that a named compound datatype that refers to a named
+ *          string datatype can be reopened indirectly through H5Dget_type,
+ *          and shows the correct H5Tcommitted() state, including after the
+ *          file has been closed and reopened.
+ *
+ * Return:  Success:    0
+ *          Failure:    number of errors
+ *
+ * Programmer:  Mark Hodson
```

Comment:
We no longer include "Programmer:" lines

Reply:
Removed.